vagrant/puppet/cmf/manifests/application/parameters.pp

```diff
@@ -0,0 +1,21 @@
+class cmf::application::parameters(
+  $dist_template = undef
+) {
+  if $dist_template != undef {
+    file { '/var/www/cmf/app/config/parameters.yml.dist':
+      ensure  => present,
+      content => template($dist_template),
+    }
+
+    file { '/var/www/cmf/app/config/parameters.yml':
+      ensure  => present,
+      content => template($dist_template),
```

Comment:
the template content should be parsed and passed into a variable. Parsing two times it not necessary